spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniparc/mapper/UniParcSourceJoin.java

```diff
@@ -0,0 +1,39 @@
+package org.uniprot.store.spark.indexer.uniparc.mapper;
+
+import java.io.Serial;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.spark.api.java.Optional;
+import org.apache.spark.api.java.function.PairFunction;
+
+import scala.Tuple2;
+
+public class UniParcSourceJoin
```

Comment:
We should rename this class to capture the intent

1. UniProtToUniParcSouceMapper
2. AccessionToUniParcIdSourceTransformer
3. AccessionToUniParcIdSourceConverter